commands/cleanup.go

```diff
@@ -0,0 +1,43 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/MichaelMure/git-bug/entities/bug"
+	"github.com/MichaelMure/git-bug/entities/identity"
+
+	"github.com/spf13/cobra"
+
+	"github.com/MichaelMure/git-bug/commands/execenv"
+)
+
+func newCleanUpCommand() *cobra.Command {
+	env := execenv.NewEnv()
+
+	cmd := &cobra.Command{
+		Use:     "cleanup",
```

Comment:
Let's do `wipe`: https://github.com/MichaelMure/git-bug/issues/676#issuecomment-1329077325

Reply:
Updated, now it's `wipe`.